var/spack/repos/builtin/packages/z3/package.py

```diff
@@ -20,8 +20,11 @@ class Z3(MakefilePackage):
 
     phases = ['bootstrap', 'build', 'install']
 
-    variant('python', default=False, description='Enable python support')
-    depends_on('python', when='+python')
+    variant('python', default=False, description='Enable python binding')
+    depends_on('python')
+
+    # Referenced: https://github.com/Z3Prover/z3/issues/1016
+    patch('fix_1016.patch')
```

Comment:
This patch works for all versions of z3 in Spack?

Reply:
I'm sorry, the patch could not be applied to z3 of ver.4.4.1 or earlier. Therefore, a new patch for ver.4.4.1 or earlier was added.